.changeset/red-mirrors-tap.md

```diff
@@ -0,0 +1,6 @@
+---
+"@medusajs/cart": patch
```

Comment:
are we adding the new modules already in the changset?

Reply:
Removed, good catch